Docs/Conventions/CodeStyle_CSharp.md

```
@@ -0,0 +1,23 @@
+| 대상 | 규칙 | 예시 |
+|---|---|---|
+| 클래스/구조체/열거형/속성/메서드 | PascalCase | ChatManager, MessageType, LoadConfig |
+| 인터페이스 | IPascalCase | IChatService |
+| 상수(const) | UPPER_SNAKE_CASE | DEFAULT_TIMEOUT_MS |
+| 비공개 필드 | _camelCase | _sessionId |
+| 직렬화 비공개 필드 | _camelCase | _gain |
+| 매개변수/지역변수 | camelCase | messageText |
+| 제네릭 매개변수 | TName | TItem, TResponse |
+
+## C# 패턴
+| 항목 | 규칙 | 예시 |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)